README.md

```diff
@@ -238,3 +238,43 @@ Notes:
 * This API is not defined by O2ims Interface Specification.
 * The server supports the `alarmProbableCauses` endpoint for exposing a custom list of probable causes.
 * The list is available in [data folder](internal/files/alarms/probable_causes.json). Can be customized and maintained as required.
+
+#### Alarm Subscription server
+
+The alarm subscription server needs to connect to the non-kubernetes API of the
+ACM global hub. To use the configmap to persist the subscriptions, the namespace "orantest" should be created at hub cluster for now.
+
+Start the deployment manager server with a command like this:
+
+```
+$./oran-o2ims start alarm-subscription-server \
+--log-file="servers.log" \
+--log-level="debug" \
+--log-field="server=alarm-subscription" \
+--log-field="pid=%p" \
+--api-listener-address="127.0.0.1:8006" \
+--metrics-listener-address="127.0.0.1:8008" \
```

Comment:
Not related to this PR, but I guess worth adding this also to the other examples in this readme later.

Reply:
Add to other examples as well.